Packs/Zoom/Integrations/Zoom/Zoom.py

```diff
@@ -1590,11 +2052,9 @@ def zoom_list_messages_command(client, **args) -> CommandResults:
             error_message = e.message
             if 'The next page token is invalid or expired.' in error_message and next_page_token:
                 raise DemistoException(f"Please ensure that the correct argument values are used when attempting to use \
-                                       the next_page_toke.\n \
-                                       Note that when using next_page_token it is mandatory to specify date time and not \
-                                       relative time.\n \
-                                       To find the appropriate values, refer to the ChatMessageNextToken located in the context. \
-                                       \n {error_message}")
+the next_page_toke.\n Note that when using next_page_token it is mandatory to specify date time and not relative time.\n \
+To find the appropriate values, refer to the ChatMessageNextToken located in the context. \n {error_message}")
```

Comment:
Please make sure @ShirleyDenkberg goes over these English texts.
e.g. there seems to be a typo:
next_page_toke -> next_page_toke n